Dockerfile.debian

```diff
@@ -163,32 +172,29 @@ RUN cd /tmp && curl -L https://sourceforge.net/projects/zsh/files/zsh/5.5.1/zsh-
     sudo echo "/usr/local/bin/zsh" >> /etc/shells && \
     cd .. && rm -fr zsh-*
 
+# install python-based utils and libs
+COPY files/python-requirements/** /tmp/python-requirements/
+RUN cd /tmp && \
+    curl https://bootstrap.pypa.io/get-pip.py -o get-pip.py && \
+    python2 get-pip.py && \
+    python3 get-pip.py && \
+    rm get-pip.py && \
+    pip2.7 install -U setuptools && \
+    pip3.5 install -U setuptools && \
+    pip2.7 install -r /tmp/python-requirements/pip2.7.txt --no-cache-dir --ignore-installed six && \
+    pip3.5 install -r /tmp/python-requirements/pip3.6.txt --no-cache-dir
 
-# docker
-RUN curl -fsSL https://download.docker.com/linux/debian/gpg | sudo apt-key add - && \
-   add-apt-repository \
-     "deb [arch=amd64] https://download.docker.com/linux/debian \
-     $(lsb_release -cs) \
-     stable" && \
-   apt-get -y update && \
-   apt-get -y install docker-ce
+# Set no password for sudo users
+RUN sed -i 's@\%sudo\tALL=(ALL:ALL) ALL@\%sudo\tALL=(ALL) NOPASSWD:ALL@g' /etc/sudoers
+
+# Configure openssh-server
+RUN mkdir /var/run/sshd && \
+     sed -i 's/#Port 22/Port 2022/' /etc/ssh/sshd_config
```

Comment:
MAY: move this to provision-user scripts to allow customization.

Reply:
I think it is better to leave it where it is because the port has to match the EXPOSE statement below.